src/config.py

```diff
@@ -494,7 +548,7 @@ def initialize_config(self):
 
         yolo_cfg = self.attribute_map["yolo_config"].struct_value.fields
         self.blob_path = yolo_cfg["blob_path"].string_value
```

Comment:
[q] Do we check if blob path actually exists?

Reply:
```
8/13/2024, 12:50:43 PM error rdk.modmanager.process.local-oak-mod_/Users/seanyu/Projects/viam-camera-oak/local_run.sh.StdErr pexec/managed_process.go:280 \_ RuntimeError: Cannot load blob, file at path /Users/seanyu/Downloads/yolov8n_coco_416x416 doesn't exist.
```
DepthAI checks for us. But we could do it. I can update

Reply:
Done

Reply:
Doesn't seem necessary to add an extra check, but, where would we expect the error to bubble up if not found?

Reply:
It bubbles up from DepthAI in our robot logs

Reply:
ok, manual check looks good.